static/js/stream_list.js

```diff
@@ -269,6 +274,7 @@ exports.create_sidebar_row = function (sub) {
         return;
     }
     build_stream_sidebar_row(sub);
+    $('.stream-name[data-toggle="tooltip"]').tooltip();
```

Comment:
right now i am unable to generate the tooltips themselves, i'm wondering what i'm doing wrong.